app/services/idv/actions/ipp/redo_address_action.rb

```diff
@@ -0,0 +1,11 @@
+module Idv
+  module Actions
+    module Ipp
+      class RedoAddressAction < Idv::Steps::DocAuthBaseStep
+        def call
+          mark_step_incomplete(:address)
```

Comment:
Do we have a plan for further changes or wiring for the IPP `RedoAddressAction` step?

Reply:
No, I think you've answered my question here.